apis/kahu/v1beta1/backup.go

```diff
@@ -19,8 +19,11 @@ import (
 )
 
 // +genclient
+// +genclient:nonNamespaced
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 // +kubebuilder:subresource:status
+// +kubebuilder:resource:scope=Cluster
+
```

Comment:
please add MetadataLocation as Required field. I had forget to add

Reply:
Issue is tracked in #50